.circleci/config.yml

```diff
@@ -41,19 +41,25 @@ jobs:
             - ./packages/core-container/node_modules
             - ./packages/core-database/node_modules
             - ./packages/core-database-postgres/node_modules
+            - ./packages/core-debugger-cli/node_modules
```

Comment:
You seem to be on an old clone as all those plugins don't exist anymore.

Reply:
Fixed